app/models/reading_collection.rb

```diff
@@ -8,7 +8,7 @@ class ReadingCollection < ApplicationRecord
   acts_as_taggable_on :tags
 
   def get_articles
-    Article.where("created_at >= ?", 5.minutes.ago).
+    Article.where("created_at >= ?", 2.minutes.ago).
```

Comment:
For demo purposes

Reply:
perfect

Reply:
Hah I've had the inverse problem testing the FE where they drop out of inclusion too quickly. But I just did a temporary local update on my end as I'm building things out.